azure-pipelines.yml

```diff
@@ -38,181 +37,190 @@ variables:
-          strategy:
-            matrix:
-              debug_configuration:
-                _BuildConfig: Debug
-                _SignType: none
-              release_configuration:
-                _BuildConfig: Release
-                _SignType: none
-          steps:
-          - ${{ if ne(variables['System.TeamProject'], 'public') }}:
-            - task: Bash@3
-              displayName: Setup Private Feeds Credentials
-              inputs:
-                filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.sh
-                arguments: $(Build.SourcesDirectory)/NuGet.config $Token
-              env:
-                Token: $(dn-bot-dnceng-artifact-feeds-rw)
-          - script: eng/common/cibuild.sh
-              --configuration $(_BuildConfig)
-              --prepareMachine
-              --integrationTest
-              $(_InternalRuntimeDownloadArgs)
-            name: Build
-            displayName: Build
+extends:
+  ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+    template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  ${{ else }}:
+    template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: $(DncEngInternalBuildPool)
+        image: 1es-windows-2022
+        os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      displayName: Build
+      jobs:
+      - ${{ if and( ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        # The localization setup for main branch. Note difference in package ID. Should not be used with release/ branches.
+        - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
+          - template: /eng/common/templates-official/job/onelocbuild.yml@self
+            parameters:
+              MirrorRepo: templating
+              LclSource: lclFilesfromPackage
+              LclPackageId: 'LCL-JUNO-PROD-TMPLTNGMAIN'
+              MirrorBranch: 'main'
+              JobNameSuffix: '_main'
+              condition: eq(variables['Build.SourceBranch'], 'refs/heads/main')
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          enableMicrobuild: true
+          enablePublishBuildArtifacts: true
+          enablePublishTestResults: true
+          enablePublishBuildAssets: true
+          enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
+          enableSourceBuild: true
+          enableTelemetry: true
+          helixRepo: dotnet/templating
+          jobs:
+          - ${{ each config in parameters.buildConfigurations }}:
+            - job: Windows_NT_${{ config.buildConfig }}
+              displayName: Windows_NT ${{ config.buildConfig }}
+              timeoutInMinutes: 90
+              pool:
+                name: $(DncEngInternalBuildPool)
+                demands: ImageOverride -equals 1es-windows-2022
```

Comment:
This shouldn't work... Need to use the 1ES PT format:
```yml
name: $(DncEngInternalBuildPool)
image: 1es-windows-2022
os: windows
```

If this does work, then my life is a lie and I'll just bury myself in the sand. ⛱️